spec/cc/engine/analyzers/python/main_spec.rb

```diff
@@ -48,19 +48,19 @@
 
       expect(json["type"]).to eq("issue")
       expect(json["check_name"]).to eq("Similar code")
-      expect(json["description"]).to eq("Similar code found in 2 other locations (mass = 6)")
+      expect(json["description"]).to eq("Similar code found in 2 other locations (mass = 5)")
       expect(json["categories"]).to eq(["Duplication"])
       expect(json["location"]).to eq({
         "path" => "foo.py",
         "lines" => { "begin" => 1, "end" => 1 },
       })
-      expect(json["remediation_points"]).to eq(1_600_000)
+      expect(json["remediation_points"]).to eq(1_550_000)
       expect(json["other_locations"]).to eq([
         {"path" => "foo.py", "lines" => { "begin" => 2, "end" => 2} },
         {"path" => "foo.py", "lines" => { "begin" => 3, "end" => 3} },
       ])
-      expect(json["content"]["body"]).to match /This issue has a mass of 6/
-      expect(json["fingerprint"]).to eq("019118ceed60bf40b35aad581aae1b02")
+      expect(json["content"]["body"]).to match(/This issue has a mass of 5/)
+      expect(json["fingerprint"]).to eq("8941b71bb75571fca80cca37a3d23dc1")
```

Comment:
Fingerprints include mass, so I believe this is a result of the masses changing. I tried to cover what I think is the customer impact of that a bit here, though I was more preoccupied with the mass thresholds changing what's considered an issue: https://github.com/codeclimate/codeclimate-duplication/pull/164#pullrequestreview-27497512.